custom-instrumentation/attributes-events-metrics/README.md

```diff
@@ -43,3 +43,13 @@ FROM `my_app:my_event` SELECT *
 ```
 
 ![1721149915594](./image/README/1721149915594.png)
+
+### Find your custom metrics 
+
+Navigate to 'Query Your Data' in New Relic. Enter the following query. You should now see information about your custom event(s).
+
+```NRQL
+SELECT count(newrelic.timeslice.value) FROM Metric WHERE metricTimesliceName like 'Custom/%' AND entity.guid = '<guid>' FACET metricTimesliceName SINCE 30 MINUTES AGO TIMESERIES
```

Comment:
I think this is easier to read:

```suggestion
SELECT count(newrelic.timeslice.value)
FROM Metric
WHERE metricTimesliceName like 'Custom/%'
AND entity.guid = '<guid>'
FACET metricTimesliceName
SINCE 30 MINUTES AGO TIMESERIES
```

(It also will avoid introducing a horizontal scrollbar to the code block.)